<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>3. Bases de datos</title>
    <meta charset="utf-8" />
    <script src="libs/header-attrs-2.9/header-attrs.js"></script>
    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/default-fonts.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/hygge.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# 3. Bases de datos
### Licenciatura en Ciencias Genómicas,UNAM
### First version: 2021-08-22; Last update: 2021-09-20

---




&lt;style type="text/css"&gt;
/* From https://github.com/yihui/xaringan/issues/147  */
.scroll-output {
  height: 80%;
  overflow-y: scroll;
}
/* https://stackoverflow.com/questions/50919104/horizontally-scrollable-output-on-xaringan-slides */
pre {
  max-width: 100%;
  overflow-x: scroll;
}
&lt;/style&gt;


# Bases de Datos

## Contenido de la unidad


  1. [Inicios de las Bases de Datos](Clase_3_pt2_v1.0.html#4)
  
  2. [EMBL](Clase_3_pt2_v1.0.html#6)

  2. [NCBI](Clase_3_pt2_v1.0.html#8)

  3. [INSDC](Clase_3_pt2_v1.0.html#9)
  
  4. [FAIR principles](Clase_3_pt2_v1.0.html#11)

  5. [The Global Life Science Data Resources Coalition](Clase_3_pt2_v1.0.html#12)
---

## Objetivo

Conocer la historia de las bases de datos más importantes en bioinformática. 


---

## Inicios de las Base de Datos
.pull-left[
**(1965) Atlas of Protein Sequence and Structure**

Secuencias de ~70 proteínas (varias especies) con alineamientos y referencias de las secuencias.
- Búsqueda manual en publicaciones

- Verificación de secuencias

&lt;img src="imgs/clase_3_pt2/atlas.jpg" width="150px" style="display: block; margin: auto;" /&gt;

]

.pull-right[
&lt;img src="imgs/clase_3_pt2/margaretDayhoff.jpg" width="200px" style="display: block; margin: auto;" /&gt;

.small["There is a tremendous amount of information regarding evolutionary history and biochemical function implicit in each sequence and the number of known sequences is growing explosively. We feel it is important to collect this significant information, correlate it into a unified whole and interpret it."]
]


---
### Hacia un banco de secuencias

#### Rockefeller University meeting (1979)
&lt;br&gt;

**Propuesta Dayhoff**

- Conocimiento publicado es de quien lo recolecta (copyright)
	
- Cuota por suscribirse a base de datos
	
- Redistribución prohibida

&lt;br&gt;

**Propuesta Goad**

- Conocimiento publicado es de la comunidad
	
- Intercambio de secuencias sin otra retribución


---
### EMBL

#### 1980: 

- Enero → Accesibilidad para colaborar con USA

- Abril →  *“Computing and DNA Sequences”* meeting

- Junio → Anuncio de que su base de datos será de libre acceso

&lt;br&gt;

1981: 

- **primer repositorio central de secuencias (open access)**

---

## Los Alamos Sequence Database

.pull-left[
- Acordar acuerdos de cooperación con los Journals (enfoque no individual)


- Cooperación internacional con EMBL y DDBJ 


- Finales de 80s: Journals hacen obligatorio subir secuencias


- Termina en 1992: Anexado al NCBI
]

.pull-rigth[
&lt;img src="imgs/clase_3_pt2/Goad.jpg" width="225px" style="display: block; margin: auto 0 auto auto;" /&gt;
]
---
## NCBI

- 1990: BLAST

- 1991: Entrez en CD

- 1992: Anexo de GenBank

- 1993: Network Entrez

- 1994: NCBI Web (BLAST, Entrez, dbEST,  dbSTS)

- 1995: Genomes, Bankit

- 1996: OMIM

- 1997: PubMed

- 1999: Human Genome



---

## International Nucleotide Sequence Database Collaboration

.pull-left[Exchange data daily to ensure that a uniform and comprehensive collection of sequence information is available worldwide
&lt;img src="imgs/clase_3_pt2/insdc2.jpg" width="250px" style="display: block; margin: auto auto auto 0;" /&gt;
]

.pull.right[
**[INSD (the International Nucleotide Sequence Databases)](https://www.insdc.org/)**

.small[hosts, not owners]
&lt;img src="imgs/clase_3_pt2/inscd.jpg" width="350px" /&gt;
]
---

&lt;img src="imgs/clase_3_pt2/wwpdb.jpg" width="800px" style="display: block; margin: auto;" /&gt;
---
## FAIR principles

FAIR principles (Findable, Accessible (under well-defined conditions), Interoperable, and Reusable)

Antes cooperación con journal editors.

**2016** this principle of data sharing and data citation letter
.pull-left[
&lt;img src="imgs/clase_3_pt2/fair_cen.jpg" width="300px" style="display: block; margin: auto auto auto 0;" /&gt;
]
.pull-right[
&lt;img src="imgs/clase_3_pt2/FAIR.png" width="400px" style="display: block; margin: auto;" /&gt;
]
---
## The Global Life Sciences Data Resources Coalition

&lt;br&gt;

Modificar modos de financiamiento a nivel internacional

- Actualmente dependen de fondos a corto plazo

- Preveer a largo plazo 

Cooperación entre distintas instituciones

Garantizar acceso a fuentes de datos 

---

## Problemass


- [What massive database retracted papers reveals about science publishing death penalty](https://www.sciencemag.org/news/2018/10/what-massive-database-retracted-papers-reveals-about-science-publishing-s-death-penalty)


- [Retraction DataBase](http://retractiondatabase.org/)


    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="macros.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>
<style>
  .logo {
    background-image: url(logos/LCG-UNAM.png);
    background-size: contain;
    background-repeat: no-repeat;
    position: absolute;
    top: 1em;
    right: 1em;
    width: 110px;
    height: 128px;
    z-index: 0;
  }
  </style>
  
  <script>
  document
    .querySelectorAll(
      '.remark-slide-content' +
      ':not(.title-slide)' +
      // add additional classes to exclude here, e.g.
      // ':not(.inverse)' +
      ':not(.hide-logo)'
    )
    .forEach(el => {
      el.innerHTML += '<div class="logo"></div>';
    });
  </script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
